src/scene/camera.js

```diff
@@ -316,6 +319,30 @@ class Camera {
         return this._viewMat;
     }
 
+    set aperture(newValue) {
+        this._aperture = newValue;
+    }
+
+    get aperture() {
+        return this._aperture;
+    }
+
+    set sensitivity(newValue) {
+        this._sensitivity = newValue;
+    }
+
+    get sensitivity() {
+        return this._sensitivity;
+    }
+
+    set shutter(newValue) {
+        this._shutter = newValue;
+    }
+
+    get shutter() {
+        return this._shutter;
+    }
```

Comment:
Currently these functions do nothing other than se / return the value. Personally, I'd remove them, and just rename the private variables to not have underscores in them.

Reply:
This was just to have it be consistent with the other parameters, like clearDepth and clearDepthBuffer, etc.